.yarnrc.yml

```diff
@@ -4,10 +4,4 @@ enableProgressBars: false
 
 nodeLinker: node-modules
 
-plugins:
-  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
-    spec: "@yarnpkg/plugin-interactive-tools"
-  - path: .yarn/plugins/@yarnpkg/plugin-version.cjs
-    spec: "@yarnpkg/plugin-version"
-
-yarnPath: .yarn/releases/yarn-2.4.2.cjs
+yarnPath: .yarn/releases/yarn-3.0.2.cjs
```

Comment:
Why are we bundling yarn? I have never seen that in other projects. Also why can't we just use NPM that everyone already have?

Reply:
I was following this https://github.com/polkadot-js/api project to extend types and saw that other networks packages use this for yarn their workspaces projects like acala and laminar among others. Using yarn workspaces its necesary in here because api, types and type-definitions packages are cross dependencies, for instance, api uses types generated interfaces and types uses types-definitions to generate these interfaces. I was getting actually several warnings and errors with yarn 2.4.2 runing yarn install and using 3.0.2 they was resolved. This yarn workspace will not work with npm actually by the way it works and how dependecies are managed.